websites/Z/Zaiko/presence.ts

```diff
@@ -0,0 +1,138 @@
+import { iFrameData as IFrameData } from "./iframe";
+
+const presence = new Presence({
+		clientId: "1310622511419101235",
+	}),
+	getStrings = presence.getStrings({
+		playing: "general.playing",
+		paused: "general.paused",
+		watching: "general.watching",
+		watchingLive: "general.watchingLive",
+		buttonWatchStream: "general.buttonWatchStream",
+		buttonViewPage: "general.buttonViewPage",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let Data: IFrameData;
+
+presence.on("iFrameData", async (data: IFrameData) => {
+	Data = data;
+});
```

Comment:
Clarification: I meant to use the variable name `data` instead of `Data`

Reply:
Sorry, i fixed it.